test/parallel/test-crypto-fips.js

```diff
@@ -89,23 +107,23 @@ testHelper(
 testHelper(
   'stdout',
   [`--openssl-config=${CNF_FIPS_ON}`],
-  compiledWithFips() ? FIPS_ENABLED : FIPS_DISABLED,
+  FIPS_ENABLED,
```

Comment:
I don't understand how this can pass with a fips-less build of openssl.  Can you explain?